features/step_definitions/steps.rb

```diff
@@ -54,7 +54,7 @@
   visit polymorphic_path(@model)
   click_on "Edit this #{model}"
   fill_in "#{model}_name", with: 'My changed name'
-  click_on "Update #{model.titlecase}"
+  click_on "Update"
```

Comment:
Not sure if this is expected or not. I checked production and it looks like the buttons say "Update" and "Create" there as well.